app/src/main/java/org/fossasia/openevent/general/MainActivity.kt

```diff
@@ -21,6 +22,7 @@ class MainActivity : AppCompatActivity() {
     private var currentFragmentId: Int = 0
 
     override fun onCreate(savedInstanceState: Bundle?) {
+        Stetho.initializeWithDefaults(this)
```

Comment:
Build is failing. You added stetho only in the debug build but you are using it in the release build also

Reply:
@nikit19 I was asked to add stetho only to the debug section. How can I implement/use it in debug mode, can you please help ? (I cannot find how to add a code only for the debug intention)

Reply:
check the build before intializing it.

Reply:
@nikit19 Why not add the no-op version for the release build?

Reply:
Yes I meant add stetho to debug and no op for release. Use the no op version from my repository